src/current/_includes/releases/cloud/2023-07-20.md

```diff
@@ -0,0 +1,5 @@
+## July 20, 2023
+
+<h3> Console changes </h3>
+
+- The primary navigation to the **Clusters**, **Billing**, **Alerts**, and **Organization** pages in the {{ site.data.products.db }} Console is now displayed at the top of the page instead of on the left.
```

Comment:
Is it worth linking to the console login page with a "Try it out" kind of CTA?

Reply:
I don't think so, there's no real functional change